iptables/actions.go

```diff
@@ -20,6 +20,10 @@ type Action interface {
 	ToFragment(features *Features) string
 }
 
+type Referrer interface {
+	ReferredToChain() string
```

Comment:
"ReferencedChain" ?  "To" suggests "To" vs "From" to me, as that occurs a lot in our iptables handling code.